templates/partials/footers.html.jinja2

```diff
@@ -0,0 +1,6 @@
+<div class="mobile-footer">
```

Comment:
The div is missing the closing </div> tag

Reply:
good catch! I added it to the partial